silk-react-components/src/HierarchicalMapping/HierarchicalMapping.scss

```
@@ -285,6 +286,20 @@ ul.ecc-hierarchical-mapping-error-list {
     margin-right: 0.1em;
 }
 
+.ecc-silk-mapping__ruleitem-icon-green {
+    font-size: 1.39em;
+    vertical-align: bottom;
+    margin-right: 0.1em;
+    color: to_color($palette-green-800);
+}
+
+.ecc-silk-mapping__ruleitem-icon-red {
```

`.ecc-silk-mapping__ruleitem-icon-green` and `.ecc-silk-mapping__ruleitem-icon-red`can be merged together as most of the props are equal (and always will be)
```suggestion
.ecc-silk-mapping__ruleitem-icon {
    font-size: 1.39em;
    vertical-align: bottom;
    margin-right: 0.1em;
    
    &-red {
        color: to_color($palette-red-800);
    }
    &-green {
        color: to_color($palette-green-800);
    }
```

Pls check before that `.ecc-silk-mapping__ruleitem-icon` not already exists